packages/cli/src/credentials/credentials.controller.ee.ts

```diff
@@ -134,50 +134,56 @@ EECredentialsController.post(
  * Grant or remove users' access to a credential.
  */
 
-EECredentialsController.put('/:credentialId/share', async (req: CredentialRequest.Share, res) => {
-	const { credentialId } = req.params;
-	const { shareWithIds } = req.body;
+EECredentialsController.put(
+	'/:credentialId/share',
+	ResponseHelper.send(async (req: CredentialRequest.Share, res) => {
```

Comment:
looks like we can't mix `ResponseHelper.send` and `res.send` as they each try to set their own set of headers, which fails.

Reply:
Yup, I messed up, pushing new changes now